demo/scripts/controls/MainPane.scss

```diff
@@ -16,6 +16,12 @@
 }
 
 .editor {
+    :global(table._tableSelected td *::selection) {
+        background-color: transparent;
+    }
+    :global(table._tableSelected td::selection) {
+        background-color: transparent;
```

Comment:
Does it mean any project that use roosterjs need to add this css to their page?

Reply:
This is only when the selection starts outside of a table and moves in a table, for selection inside a single table this is not taking effect
Next gif is without this css rules.
So it is not required to add this to any project css.

If the :selection and the cell style we are applying to cells is to noisy, we can remove the code that handle highlighting cells when the selection is outside of a table
![TABL1](https://user-images.githubusercontent.com/8291124/150146028-157f5031-9cce-4830-83ce-71c332ba1c55.gif)